kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -39,7 +62,11 @@ export async function spawnPortableExperienceScene(portableExperienceUrn: Portab
     icon: scene.data.data.icon,
     isPortableExperience: true
   })
-  return parcelSceneId
+
+  const contentId = cid ?? ''
```

Comment:
In which cases this is ""? Why do we support it instead of throwing an error?

Reply:
Currently, all CID must be present, so the best would be to throw an error.

Reply:
Currently, we are starting the PE in the main, that's why we didn't have a parent process.  I'll put the parentCid as mandatory and hardcode it in the main call, as it will be removed after testing.